Lib/fontTools/feaLib/lexer.py

```diff
@@ -5,7 +5,6 @@
 import re
 import os
 
-
```

Comment:
Correct, and there is no need for it. Is is important to have 3 empty lines there? Is that part of your coding standard.